### DISCUSSION TITLE
Enable Plugin on 'Photos' Library

### DISCUSSION THREAD

**Discussion by moduraziel** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#51] [Ideas]
Due to the way that I have all my porn organized and subfoldered, the 'Photos' content type is the best one to use in order for things to display correctly. The issue is that this plugin is not available for that content type. Is it possible to get it added as a metadata downloader for 'photos' libraries? Note: I am not trying to gather metadata on images but on the video files that are in the folders alongside images

**Comment by DirtyRacer1337** [comment]
It's possible in theory, but I don't want to add that. Too much needs to be changed for that.

**Reply by moduraziel** [reply]
Why would anything need to change? Just enable it for additional library types. Literally all the functionality would be the same

**Reply by DirtyRacer1337** [reply]
That's not working so easy. Right now it's returning only movie type, I can't just return all possible types. I need find time to implement that.

**Reply by moduraziel** [reply]
I stress again that I am NOT trying to return metadata on images, I am trying to download movie metadata. The video files are just under the Photos content type in Jellyfin because that is the only library option that allows for showing image files alongside video files.

**Reply by DirtyRacer1337** [reply]
I'm talking about Library type, not content.
